src/components/WalletTypeButton.tsx

```diff
@@ -37,16 +37,24 @@ export default function WalletTypeButton({
       >
         <Image
           src={isActive ? content.activeImage : content.inactiveImage}
-          width={96}
-          height={71}
+          width={120}
+          height={120}
           alt="Mens Bags"
-          style={{ height: "100%", width: "auto" }}
+          style={{ height: 100, width: 115 }}
         />
-        <div className="absolute group-hover:text-orange-600 text-2xl flex justify-center translate-y-32">
+
+        <div
+          className={`absolute group-hover:text-orange-600 text-2xl flex justify-center translate-y-32 ${
+            isActive ? `text-orange-600` : `text-black`
+          }`}
+        >
           <BsChevronCompactDown />
         </div>
-
-        <div className="group-hover:text-orange-600 text-sm translate-y-4 text-gray-400">
+        <div
+          className={`group-hover:text-orange-600 text-sm translate-y-4 ${
+            isActive ? "text-orange-600" : "text-gray-400"
```

Comment:
Could extract isActive class and be used in both locations

Reply:
addressed.